add slack docs (#21)
team-plain/docs: slack.mdx

---
title: 'Slack'
sidebarTitle: 'Overview'
---

**Never miss another message from a customer in Slack**

Our Slack integration lets you sync messages from selected Slack channels to Plain and respond directly to customers from the platform.

You will also be able to [add context to Slack messages with customer cards](/customer-cards), [set labels](/labels) and priorities, [create Linear tickets](/linear), and much more.

Here’s how to get started:

**1 Connect your workspaces**

- In Plain, navigate to **`Settings`** > **`Slack`**.
- Press **`Connect to Slack`**.

> - Follow Slack’s instructions making sure the correct Slack workspace is selected.

Reviewer: ```suggestion
- Follow Slack’s instructions to make sure the correct Slack workspace is selected.
```

- Authorize replies from your Slack profile to respond directly from Plain.

**2 Add the Plain app to the channels you want to track**

Once you’ve set up the integration in Plain, go to the Slack channels you want to track and invite the Plain app. You can do this by typing **`/invite`**, selecting **`Add apps to this channel`** and then choosing Plain.

<video
  controls
  className="w-full aspect-video"
  src="https://static-assets-git-add-slack-video-team-plain.vercel.app/docs/slack/add-to-slack.mp4"
></video>

> **3 Start responding 💬**

Reviewer: ```suggestion
**3 Start responding **
```

> Slack messages from the authorized channels will begin appearing as threads in Plain. Press `R` and begin typing in Plain. Your responses will appear in Slack as if you are responding directly in the app.

Reviewer: ```suggestion
Slack messages from the authorized channels will begin appearing as threads in Plain. To respond, press `R` and begin typing in Plain. Your responses will appear in Slack as if you are responding directly in the app.
```
